### ISSUE TITLE
normalizedRequestAttrs stopped working for dasherized paths

### ISSUE THREAD

**Issue by wuron** [samselikoff/ember-cli-mirage#1440]
Hi there.
There's a bug with normalizedRequestAttrs in 0.4.10 version of mirage that I believe was caused by this commit: https://github.com/samselikoff/ember-cli-mirage/commit/4606c11ab8480698a6ed42e819b645967e3562f3

Previously having `app/models/foo-bar.js` defined I could do something like this in my mirage/config.js

```js
export default function () {
  this.patch('foo-bars/:id', function () {
    const attrs = this.normalizedRequestAttrs();
  });
}
```

Now this doesn't work because of added assertion `this.schema.modelFor(modelName)` in `normalizedRequestAttrs`. See https://github.com/samselikoff/ember-cli-mirage/commit/4606c11ab8480698a6ed42e819b645967e3562f3#diff-9ff4e2a84eb4c76ed9a4d8c6d117fda4R48

Also it looks like this assertion is not necessary, because the `_getAttrsForRequest` function actually uses `this.schema.modelClassFor(modelName)` which camelizes model name.

So the solution seems to either assert for `this.schema.modelClassFor(modelName)` instead of `this.schema.modelFor(modelName)` in `BaseRouteHandler`, or to modify `modelFor` method in schema so that it also camelizes its argument to be consistent with `modelClassFor`.

**Comment by mydea** [comment]
fyi, I was able to "fix" this by manually specifying the model name in camel-cased, singular form, e.g.

```
this.normalizedRequestAttrs('fooBar');
```

**Comment by barryofguilder** [comment]
I noticed this today too after upgrading. I would have though that the `modelName` for `normalizedRequestAttrs` would be expecting a dasherized name like we do when calling `server.create`, etc.

**Comment by samselikoff** [comment]
it should definitely be dasherized. Fixing this now.

**Comment by samselikoff** [comment]
@wuron @barryofguilder @mydea Just released [0.4.11](https://github.com/samselikoff/ember-cli-mirage/releases/tag/v0.4.11), could y'all please give it a try and let me know if it fixes the issue?

**Comment by barryofguilder** [comment]
@samselikoff I just updated my project and all my tests are passing, so this fixed my issue!
